content/events/members/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Members"
+date: 2022-02-04T10:00:45-04:00
+hide_sidebar: true
+layout: "single"
+---
+
+{{< grid/div isMarkdown="false">}}
+  <p class="margin-0">Are you attending an event you want the OpenHW community to know about?</p>
+  <p><a href="https://newsroom.eclipse.org/node/add/events" target="_blank">Submit your event</a> details, and indicate that there is a "Member organization participating", to appear on this page.</p>
+{{</ grid/div >}}
+
+{{< newsroom/events
+    id="events_archive" 
+    archive="true"
+    publishTarget="openhwgroup&parameters[ef_participation][]=member_org_participating"
```

Comment:
Hi @chrisguindon , this is what I talked about in the meeting. For a quick and dirty implementation, I just used the "publishTarget" for "ef_participation". 

Let me know if we prefer creating a new filter in our short code and solstice-assets. 
If so, what I was thinking is to create a flexible filter options like "customFilter". This way, we won't need to create or update the shortcode and soltsice-assets when we need to use a new filter option. And we can still add specific filter options in if it's a very commonly used one. 

WDYT?

Reply:
Sounds good! Created the issue: https://gitlab.eclipse.org/eclipsefdn/it/webdev/solstice-assets/-/issues/290